wiki/Tournaments/OMIC/2022_4K/id.md

```diff
@@ -0,0 +1,420 @@
+| Winner | **Fr05tyGD** ::{ flag=ID }:: | **0** | -1 | ::{ flag=ID }:: \[ -Asriel- \] | [WBD]() |
+| Winner | **Dellia** ::{ flag=ID }:: | **5** | 1 | ::{ flag=ID }:: LuxSayaka | [#1](https://osu.ppy.sh/community/matches/102341106) |
+
+### Babak 32 Besar
+
+Sabtu, 9 Juli 2022
+
+| Bracket | Player 1 |  |  | Player 2 | Match link |
+| :-: | --: | :-: | :-: | :-- | :-- |
+| Winner | **RetroEX** ::{ flag=ID }:: | **4** | 1 | ::{ flag=ID }:: Yururu | [#1](https://osu.ppy.sh/community/matches/102122568) |
+| Winner | **adihza** ::{ flag=ID }:: | **0** | -1 | ::{ flag=ID }:: MzLord | [WBD](https://osu.ppy.sh/community/matches/102126904) |
+| Winner | **TheLegend3RD** ::{ flag=ID }:: | **0** | -1 | ::{ flag=ID }:: EVIRE- | [WBD](https://s3.amazonaws.com/challonge_app/match_attachments/assets/000/730/058/) |
+| Winner | **-Yubi-** ::{ flag=ID }:: | **4** | 2 | ::{ flag=ID }:: spectator\_boi | [#1](https://osu.ppy.sh/community/matches/102127806) |
+| Winner | Tsurumaki\_ ::{ flag=ID }:: | 1 | **4** | ::{ flag=ID }:: **Heriza-** | [#1](https://osu.ppy.sh/community/matches/102131078) |
+| Winner | **Dellia** ::{ flag=ID }:: | **4** | 0 | ::{ flag=ID }:: danar mw | [#1](https://osu.ppy.sh/community/matches/102130738) |
+| Winner | **\[ReyZ\]** ::{ flag=ID }:: | **0** | -1 | ::{ flag=ID }:: Dogethedoge25 | [WBD](https://osu.ppy.sh/community/matches/102130389) |
+| Winner | **AZKiFanboy** ::{ flag=ID }:: | **0** | -1 | ::{ flag=ID }:: AzusaNakano2 | [WBD](https://osu.ppy.sh/community/matches/102152878) |
+
+Minggu, 10 Juli 2022
+
+| Bracket | Player 1 |  |  | Player 2 | Match link |
+| :-: | --: | :-: | :-: | :-- | :-- |
+| Winner | Revv- ::{ flag=ID }:: | -1 | **0** | ::{ flag=ID }:: **LuxSayaka** | [WBD](https://s3.amazonaws.com/challonge_app/match_attachments/assets/000/730/061/) |
+| Winner | **Onta\_Bekasi** ::{ flag=ID }:: | **0** | -1 | ::{ flag=ID }:: KyoukaRy | [WBD](https://s3.amazonaws.com/challonge_app/match_attachments/assets/000/730/350/) |
+| Winner | **haickalmh** ::{ flag=ID }:: | **4** | 1 | ::{ flag=ID }:: Lordkolp | [#1](https://osu.ppy.sh/community/matches/102151359) |
+| Winner | **Fieristik** ::{ flag=ID }:: | **4** | 1 | ::{ flag=ID }:: KinoJourney | [#1](https://osu.ppy.sh/community/matches/102152651) |
+| Winner | **Fr05tyGD** ::{ flag=ID }:: | **4** | 0 | ::{ flag=ID }:: HorizonID | [#1](https://osu.ppy.sh/community/matches/102154216) |
+| Winner | **\[ -Asriel- \]** ::{ flag=ID }:: | **0** | -1 | ::{ flag=ID }:: CrossModic | [WBD](https://s3.amazonaws.com/challonge_app/match_attachments/assets/000/730/063/) |
+| Winner | **HEAL\_69** ::{ flag=ID }:: | **4** | 2 | ::{ flag=ID }:: Proxae | [#1](https://osu.ppy.sh/community/matches/102156827) |
+| Winner | **\[Albert\]** ::{ flag=ID }:: | **4** | 1 | ::{ flag=ID }:: \[ Albedo \] | [#1](https://osu.ppy.sh/community/matches/102156682) |
+
+## Peraturan Umum
```

Comment:
this header for the Rules section here appears as if it's out of place tbh, would recommend sth like

`## Peraturan`

`### Ketentuan Pendaftaran`

`### Peraturan Umum`

`### Regulasi Pertandingan`

Reply:
will be applied in a separate commit. Thank you.